frontend/src/test/scala/bloop/HydraCompileSpec.scala

```diff
@@ -13,9 +13,14 @@ import scala.tools.nsc.Properties
 
 object HydraCompileSpec extends BaseCompileSpec {
   override protected val TestProject = HydraTestProject
+
   override protected def extraCompilationMessageOutput: String =
     " [E-1] Using 1 Hydra worker to compile Scala sources."
 
+  override protected def processOutput(message: String) = {
+    message.replace(" [E-1] License will expire in 14 days.\n", "")
```

Comment:
What if we run the test tommorrow? Will the message contain `13 days` rather than `14 days`?

Reply:
Good point :S I didn't think about it